build-config/src/main/kotlin/configuration/configure-publishing.gradle.kts

```diff
@@ -0,0 +1,49 @@
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+import com.vanniktech.maven.publish.SonatypeHost
+import extensions.isMultiplatformModule
+import extensions.kotlinMultiplatform
+
+pluginManager.apply("com.vanniktech.maven.publish")
+
+if (isMultiplatformModule()) {
+    kotlinMultiplatform {
+        androidTarget {
+            publishLibraryVariants("release")
+        }
+    }
+}
+
+group = "cafe.adriel.voyager"
+
+configure<MavenPublishBaseExtension> {
+    publishToMavenCentral(host = SonatypeHost.S01, automaticRelease = true)
+    signAllPublications()
+
+    pom {
+        description = "A pragmatic navigation library for Jetpack Compose"
+        inceptionYear = "2021"
+        url = "https://github.com/adrielcafe/voyager"
+
+        licenses {
+            license {
+                name = "The MIT License"
+                url = "https://opensource.org/licenses/MIT"
+                distribution = "repo"
+            }
+        }
+
+        scm {
+            url = "https://github.com/adrielcafe/voyager"
+            connection = "scm:git:ssh://git@github.com/adrielcafe/voyager.git"
+            developerConnection = "scm:git:ssh://git@github.com/adrielcafe/voyager.git"
+        }
+
+        developers {
+            developer {
+                id = "adrielcafe"
+                name = "Adriel Cafe"
+                url = "https://github.com/adrielcafe/"
+            }
+        }
+    }
+}
```

Comment:
Why this was required? why not keep it on `gradle.properties`?

Reply:
I think explicit configuration is better than implicit